packages/flutter_adaptive_scaffold/example/lib/adaptive_layout_demo.dart

```diff
@@ -154,7 +154,7 @@ class MyHomePage extends StatelessWidget {
               extended: true,
               // #enddocregion Example
               // TODO(goderbauer): Make this const when this package requires Flutter 3.8 or later.
-              // ignore: prefer_const_constructors
+              // ignore_for_file: prefer_const_constructors
```

Comment:
If you're going to ignore it for the file, can you put the comment at the top of the file so it's easier to notice?

Or, preferably, if there's only one or two other instances, just ignore for each instance.

Reply:
Oh, actually, it looks like the ignore should just be removed. See [here](https://github.com/flutter/packages/pull/3174/checks?check_run_id=11148500216)

Reply:
Three options:
ignore for file, remove it from the todo and put it at the top,
ignore for file, leave it in the todo,
ignore for line, add it to the `docregion` and then into the readme as well.

Happy to do whichever you think is best.

Reply:
No, I think you just remove the ignore entirely, since it appears to be a duplicate.  The error message is: 

```
warning - example/lib/adaptive_layout_demo.dart:49:20 - The diagnostic 'prefer_const_constructors' doesn't need to be ignored here because it's already being ignored. Try removing the name from the list, or removing the whole comment if this is the only name in the list. - duplicate_ignore
```

Reply:
Sorry, I added the manual roll to this pr now as well. It requires this change

Reply:
oops, there were 2! Fixed

Reply:
Ahh, that's better. LGTM (again)